Dockerfile

```diff
@@ -92,6 +92,11 @@ RUN mkdir -p ${HOME}/.config/matplotlib && \
     echo "backend      : Agg" > ${HOME}/.config/matplotlib/matplotlibrc && \
     python -c "import matplotlib.pyplot"
 
+# Enable widgetsnbextension for jupyter widgets.
+# See https://ipywidgets.readthedocs.io/en/stable/user_install.html.
+# This enables the extension in the conda environment.
+RUN jupyter nbextension enable --py widgetsnbextension
```

Comment:
Is this still necessary if you explicitly install the ipywidgets package, instead of letting it be pulled in as a dependency? The documentation you link to says this command shouldn't be needed when installing from conda.

Reply:
Random comment: I have also found that you do not need the command above when installing from conda locally. Conda has a post-link feature which allows for stuff like this after packages are installed.